src/iop/finalscale.c

```diff
@@ -148,6 +150,22 @@ void cleanup(dt_iop_module_t *self)
   self->default_params = NULL;
 }
 
+typedef struct dt_iop_finalscale_gui_data_t
+{ } dt_iop_finalscale_gui_data_t;
+
+dt_iop_finalscale_gui_data_t dummy;
+
+void gui_init(dt_iop_module_t *self)
+{
+  IOP_GUI_ALLOC(finalscale);
+  self->widget = gtk_label_new(NULL);
+  gtk_label_set_markup(GTK_LABEL(self->widget),_("This module is used to downscale images at export time. "
+                                                 "Moving it along the pipeline will have diffent effects on exported images. "
+                                                 "<a href='https://ansel.photos/en/doc/modules/processing-modules/finalscale/'>Learn more</a>"));
```

Comment:
We should refer to some dt manual section here :-)

Reply:
since it takes time to prepare the manuals i'd prefer to ditch that link ;)